src/04-modality-specific-files/11-functional-near-infrared-spectroscopy.md

```diff
@@ -0,0 +1,404 @@
+    In these cases additional information regarding channels and optodes SHOULD be placed in `*_channels.tsv` and `*_optodes.tsv` files.
+
+Closely spaced or short-separation source-detector pairs are often included in fNIRS measurements to
+obtain a measure of systemic, rather than neural, activity. These source-detector
+pairs are referred to as *short channels*. There is variation in how manufacturers
+implement these short channels, some use specialised sources or detectors,
+and the placement mechanisms vary.
+It is beyond the scope of the BIDS specification to define what constitutes a short channel,
+and detailed characteristics of channels may be stored within the SNIRF file
+(for example, in the `sourcePower` field).
+However, to improve searchability and ease of access for users, it is useful to
+know if short channels were included in the fNIRS measurements; the presence of short channels is
+is stored in the field `ShortChannelCount`.
+If the field `ShortChannelCount` is populated, then the optional column `short_channel`
+may be used in `*_channels.tsv` to describe which channels were specified as short.
+
+Generic fields that MUST be present:
+
+{{ MACROS___make_metadata_table(
+   {
+      "TaskName": "REQUIRED",
+   }
+) }}
+
+Generic fields which SHOULD be present: For consistency between studies and institutions,
+we encourage users to extract the values of these fields from the actual raw data.
+Whenever possible, please avoid using ad hoc wording.
+
+{{ MACROS___make_metadata_table(
+   {
+      "InstitutionName": "RECOMMENDED",
+      "InstitutionAddress": "RECOMMENDED",
+      "Manufacturer": "RECOMMENDED",
+      "ManufacturersModelName": "RECOMMENDED",
+      "SoftwareVersions": "RECOMMENDED",
+      "TaskDescription": "RECOMMENDED",
+      "Instructions": "RECOMMENDED",
+      "CogAtlasID": "RECOMMENDED",
+      "CogPOID": "RECOMMENDED",
+      "DeviceSerialNumber": "RECOMMENDED",
+      "RecordingDuration": "RECOMMENDED",
+      "HeadCircumference": "RECOMMENDED",
+      "HardwareFilters": "RECOMMENDED",
+      "SubjectArtefactDescription": "RECOMMENDED",
+   }
+) }}
+
+{{ MACROS___make_metadata_table(
+   {
+      "CapManufacturer": ("RECOMMENDED", "If a custom-made cap is used then the string `custom` should be used. If no cap was used, such as with optodes that are directly taped to the scalp, then the string `none` should be used and the `NIRSPlacementScheme` field may be used to specify the optode placement."),
+      "CapManufacturersModelName": ("RECOMMENDED", "If there is no official model number then description may be provided (for example, `Headband with print (S-M)`). If a cap from a standard manufacturer was modified, then the field should be set to `custom`. If no cap was used, then the `CapManafacturer` field should be `none` and this field should be `n/a`."),
+   }
+) }}
+
+Specific fNIRS fields that MUST be present:
+
+TODO: In the table below the SamplingFrequency data type should be "number or n/a".
```

Comment:
@sappelhoff do you know how I can modify SamplingFrequency to be "number or n/a" for NIRS, but not change the type for the other modalities?

Reply:
This could be tricky schema-wise. In the past we have sometimes created the same schema entry twice with a small hack. for example the metadata `name` has a different meaning when used in a channels.tsv or an electrodes.tsv. See for example: https://github.com/bids-standard/bids-specification/blob/9201b203ffaa72d83b2fa30d1c61f46f089f77de/src/schema/objects/columns.yaml#L202-L219

I think for `SamplingFrequency`, we'd either have to do something similar (please advise, @tsalo  :-) ) ... or we could discuss making `n/a` an admissible option for `SamplingFrequency` throughout the spec.

I would be open to that if users then specify the sampling frequency in the `sampling_frequency` column in their `channels.tsv` file (as you also suggest in your BEP). Not having any specification of sampling frequency should lead to a validator error. -- Having that said, the only problem I see with that route would be implementing support for that in the validator not only for fnirs, but also meg, eeg, and ieeg cc @rwblair

Reply:
Thanks. I have implemented `SamplingFrequency__nirs` and the doc renders correctly. We can change to another solution if required later.